app/src/main/java/com/github/sdp_begreen/begreen/FirebaseDB.kt

```diff
@@ -32,7 +53,73 @@ class FirebaseDB : Database() {
     /**
      * Create a new tuple ([key], [value]) if [key] was not present and update the value otherwise
      */
-    override fun set(key: String, value: String) {
-        firebaseDB.child(key).setValue(value)
+    operator fun set(key: String, value: String) {
+        databaseReference.child(key).setValue(value)
+    }
+
+    /**
+     * Store the given [image] under the ID [userId]
+     * and returns the imageId set for that image in the database
+     * Returns null if the image can't be stored
+     */
+    fun addImage(image: Bitmap, userId: Int): String? {

```

Comment:
That's a good point, there are indeed other functionalities provided by Firebase that are more suited to store images. Since this (hopefully) will not change the interface of the FirebaseDB class, I'll make this change later, since Bryan is waiting for this merge advance. Thanks for pointing that out :)

Reply:
Yes indeed, storing images in the realtime database seems (and is) painful, I just thought I was meant to use that and not firestore or anything else. Since this should not modify the interface of the "FirebaseDB" methods, I'll keep it that way for the moment and change later since Bryan needs the implementation of "FirebaseDB" for one of his task, and he's waiting on me. I'll definitly apply changes soon